src/easydmp/plan/models.py

```diff
@@ -618,16 +629,13 @@ def validate_data(self, recalculate=True):
             LOG.error(error.format(self, self.pk))
             return False
         if recalculate:
-            for sv in self.answersets.all():
-                valids, invalids = sv.section.find_validity_of_questions(self.data)
-                self.set_validity_of_questions(valids, invalids)
-            valids, invalids = self.template.find_validity_of_sections(self.data)
-            self.set_validity_of_sections(valids, invalids)
+            for answerset in self.answersets.all():
+                answerset.validate(_data=self.data)
         if self.answersets.filter(valid=True).count() == self.template.sections.count():
             return True
         return False
 
-    def validate(self, user, recalculate=False, commit=True, timestamp=None):
+    def validate(self, user: Any, recalculate: bool = False, commit: bool = True, timestamp: datetime = None) -> None:
```

Comment:
`user: Any` should be `user: User`, possibly imported via a type checking guard.

Reply:
The easydmp.auth one I presume.